scalafix-core/src/main/scala/scalafix/rewrite/ExplicitReturnTypes.scala

```diff
@@ -20,20 +23,40 @@ case class ExplicitReturnTypes(mirror: Mirror)
 
   def defnName(defn: Defn): Option[Name] = Option(defn).collect {
     case Defn.Val(_, Seq(Pat.Var.Term(name)), _, _) => name
+    case Defn.Var(_, Seq(Pat.Var.Term(name)), _, _) => name
     case Defn.Def(_, name, _, _, _, _) => name
   }
 
+  def visibility(mod:Mod):Option[MemberVisibility] = Option(mod).flatMap{
+    case _:Mod.Private => Some(MemberVisibility.Private)
+    case _:Mod.Protected => Some(MemberVisibility.Protected)
+    case _ => None
+  }
+
+  def kind(defn:Defn):Option[MemberKind] = Option(defn).flatMap{
+    case _:Defn.Val => Some(MemberKind.Val)
+    case _:Defn.Def => Some(MemberKind.Def)
+    case _:Defn.Var => Some(MemberKind.Var)
+    case _          => None
+  }
+
   def parseDenotationInfo(denot: Denotation): Option[Type] = {
     val base =
       if (denot.isVal) denot.info
+      else if(denot.isVar) denot.info.replaceFirst("var ", "")
       else if (denot.isDef) denot.info.replaceFirst(".*\\)", "")
       else {
         throw new UnsupportedOperationException(
           s"Can't parse type for denotation $denot, denot.info=${denot.info}")
       }
-    base.parse[Type].toOption
+    if(denot.isVal || denot.isDef)
+      base.parse[Type].toOption
+    else
+      base.parse[Stat].toOption.flatMap{_.collect{case Term.Ascribe(_,typ) => typ}.headOption}
```

Comment:
This is added to extract a "argument" type from dennotation of Var like `var c_= (x$1 : Int):Unit`
I would appreciate it if you can suggest a smarter way!